include/ck/tensor_operation/gpu/grid/gridwise_batched_mha_bwd_xdl_cshuffle_qloop_b2t_v1.hpp

```diff
@@ -537,16 +564,44 @@ struct GridwiseBatchedMultiheadAttentionBackward_Qloop_Xdl_CShuffle_V1
             make_multi_index(0, -MPerBlock, 0);
     };
 
-    // S / dP Gemm (type 1 rcc)
+    // dP Gemm (type 1 rcc)
     struct Gemm0
     {
         // A matrix in LDS memory, dst of blockwise copy
         static constexpr auto a_block_desc_ak0_m_ak1 =
             GetABlockDescriptor_AK0PerBlock_MPerBlock_AK1();
 
-        // B matrix in LDS memory, dst of blockwise copy
-        static constexpr auto b_block_desc_bk0_n_bk1 =
-            GetBBlockDescriptor_BK0PerBlock_NPerBlock_BK1();
+        // B source matrix layout in VGPR
+        static constexpr auto b_src_thread_desc_k0_k1_n0_n1_n2_n3_k2 =
+            GetVThreadDescriptor_K0_K1_N0_N1_N2_N3_K2();
```

Comment:
Move this to be template parameter of `struct Gemm0`,  don't use `V` here inside `struct Gemm0`